### ISSUE TITLE
[Suggestion] Add "Co-op team simulator" and adding skill inheritance feature.

### ISSUE THREAD

**Issue by TheBaddest** [madcowfred/padherder-issues#914]
More multiplayer content is coming and I believe this should be really helpful. PADX doesn't have this feature and recently even their regular team simulator isn't working properly.

The idea is: a user can visit another user's "My Teams" page and use one of his teams from a drop down list to make a coop team simulator. The general layout should be similar with the current "my teams" layout, but with both teams from both users shown side by side. Total stats and awakenings list is listed at the bottom.

To add to that, adding an "assist monster / skill inheritance" feature to "my monsters" list is also helpful, though isn't really important.

Thanks.

**Comment by drallyowl2** [comment]
I disagree that the assist feature is unimportant. It is an incredibly crucial aspect of several teams, and needing to tell people in the description what assist things have can ruin the ease of getting an at-a-glance idea of the team.
